crds/appcat.vshn.io_objectbuckets.yaml

```diff
@@ -60,6 +60,16 @@ spec:
                 - bucketName
                 - region
                 type: object
+              writeConnectionSecretToRef:
+                description: WriteConnectionSecretToRef references a secret to which
+                  the connection details will be written.
+                properties:
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                      TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Maybe you missed that

Reply:
Hmm. Interestingly this comes from k8s's type `corev1.LocalObjectReference` :thinking: